layout-server/src/test/java/org/finos/vuu/layoutserver/integration/LayoutIntegrationTest.java

```diff
@@ -32,14 +34,15 @@
 @ActiveProfiles("test")
 public class LayoutIntegrationTest {
 
-    private static final String DEFAULT_LAYOUT_DEFINITION = "Default layout definition";
+    private static final String LAYOUT_DEFINITION_STRING = "{\"id\":\"main-tabs\"}";
```

Comment:
Why the change from `DEFAULT_LAYOUT_DEFINITION` to `LAYOUT_DEFINITION_STRING`?

Reply:
Oops! not sure how this change came about but it should probably have DEFAULT_* for consistence with the other fields. I will make the change

Reply:
I hadn't pushed the change yet as I wanted to fix the conflicts first. It's done now :)